css/style.css

```diff
@@ -0,0 +1,597 @@
+body {
```

Comment:
- [ ] You should use `#EC5242` and ` #D3D3D3` as your colors
- [ ] You should use the ` Cocogoose` and `Lato` fonts to style your text so it looks like the original design